visualizer/index.html

```diff
@@ -467,6 +470,7 @@
 
   <div id="options">
     <ul>
+      <li><input type="checkbox" name="showExampleGenerator"><label>Show Example Generator</label></li>
```

Comment:
How about "Generate examples" instead? And it should be sentence case to match the others.